### PR TITLE
Update pricing tables for Ubuntu Pro

### PR DESCRIPTION
**Pull request by carkod**
## Done

Updated pricing page for Ubuntu pro according to design
Because the design is a radical change in layout and a lot of the content, the [copy doc](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit#heading=h.l83dv5879ajz) will need to be revamped. So for this PR please use design only.

## QA

- Go to https://ubuntu-com-12268.demos.haus/pricing/pro and compare with [design](https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=2200%3A15152)

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6115
Fixes https://github.com/canonical/web-squad/issues/2341
Fixes https://github.com/canonical/ubuntu.com/issues/5343
Fixes https://github.com/canonical/ubuntu.com/issues/12137

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by carkod** [comment]
@lyubomir-popov I couldn't find some of the square logos, could you link/create if necessary? Thanks

**Comment by lyubomir-popov** [comment]
@carkod detailed QA here https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=2583%3A15341

**Comment by carkod** [comment]
> @carkod detailed QA here https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-Ubuntu-Pro?node-id=2583%3A15341

Should be ready now, let me know if I missed anything.

**Comment by lyubomir-popov** [comment]
05C441F3-174B-4510-9ACF-2DD88A726086
7806CCEB-00C3-49DB-B947-7EE6D3505166
D587EE31-7A91-4035-9DDF-C7D0B9EB3217
52E0C4D6-4226-4929-9881-AA23512F4038
23B84CF4-0977-4C00-8246-B7DC66D68B94

**Comment by carkod** [comment]
> 05C441F3-174B-4510-9ACF-2DD88A726086 7806CCEB-00C3-49DB-B947-7EE6D3505166 D587EE31-7A91-4035-9DDF-C7D0B9EB3217 52E0C4D6-4226-4929-9881-AA23512F4038 23B84CF4-0977-4C00-8246-B7DC66D68B94

Thanks. It's fixed now.

**Comment by lyubomir-popov** [comment]
hmm mobile isn't working 
image
partly because of missing th rows I think - compare with the example - but also the nesting. Maybe we should try horizontal scroll instead of the mobile cards? Ping me when you're free and we can discus @carkod

**Comment by carkod** [comment]
> hmm mobile isn't working partly because of missing th rows I think - compare with the example - but also the nesting. Maybe we should try horizontal scroll instead of the mobile cards? Ping me when you're free and we can discus @carkod

ok, yeah so I made some changes to accomodate for horizontal scrolling. Well if you've got time, we can discuss, but I think you are pretty busy this week

**Comment by carkod** [comment]
Updated with copy changes.

**Comment by carkod** [comment]
@sowasred2012 can you review when you can? We want to get this pages live asap, as current live pages are misleading. I know there is quite a bit of custom styling, but design wanted to get things on point.

File: templates/pricing/pro.html [hunk 1]
```diff
@@ -25,127 +24,542 @@ <h1>Ubuntu Pro: security, compliance, and support</h1>
...
-  <div class="u-fixed-width">
-    <div style="overflow-x: auto;">
+  <div class="row">
+    <hr class="is-fixed-width">
+    <div class="col-6">
+      <h3 class="p-heading--2">Ubuntu Pro (Infra-only) vs Ubuntu Pro</h3>
+    </div>
+
+    <div class="col-6">
+      <h5 class="u-no-margin--bottom">Ubuntu Pro (Infra-only):</h5>
```
Comment (sowasred2012) [inline templates/pricing/pro.html:132]:
```suggestion
      <h4 class="p-heading--5 u-no-margin--bottom">Ubuntu Pro (Infra-only):</h4>
```

File: templates/pricing/pro.html [hunk 2]
```diff
@@ -25,127 +24,542 @@ <h1>Ubuntu Pro: security, compliance, and support</h1>
...
       </table>
     </div>
   </div>
 
-  <div class="u-fixed-width">
-    <div style="overflow-x: auto;">
+  <div class="row">
+    <div class="col-6">&nbsp;</div>
+    <div class="col-6">
+      <h5 class="u-no-margin--bottom">Ubuntu Pro:</h5>
```
Comment (sowasred2012) [inline templates/pricing/pro.html:151]:
```suggestion
      <h4 class="p-heading--5 u-no-margin--bottom">Ubuntu Pro:</h4>
```

File: templates/pricing/pro.html [hunk 3]
```diff
@@ -25,127 +24,542 @@ <h1>Ubuntu Pro: security, compliance, and support</h1>
...
+
+<section class="p-strip is-deep u-no-padding--top">
+  <div class="row">
+    <hr class="is-fixed-width">
+    <div class="col-6">
+      <h3 class="p-heading--2">Support: Infra vs Full-stack</h3>
+    </div>
+
+    <div class="col-6">
+      <h5 class="u-no-margin--bottom"><a href="/support">Infra support</a></h5>
```
Comment (sowasred2012) [inline templates/pricing/pro.html:172]:
```suggestion
      <h4 class="p-heading--5 u-no-margin--bottom"><a href="/support">Infra support</a></h4>
```

File: templates/pricing/pro.html [hunk 4]
```diff
@@ -25,127 +24,542 @@ <h1>Ubuntu Pro: security, compliance, and support</h1>
...
+          </div>
+        </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-6">&nbsp;</div>
+    <div class="col-6">
+      <hr class="is-fixed-width">
+      <h5 class="u-no-margin--bottom"><a href="/support">Full stack support</a></h5>
```
Comment (sowasred2012) [inline templates/pricing/pro.html:273]:
```suggestion
      <h4 class="p-heading--5 u-no-margin--bottom"><a href="/support">Full stack support</a></h4>
```

**Review by sowasred2012 — APPROVED** [review]
Looks good, but please accept the suggestions before merging - minor a11y fixes.

**Pull request merged**